xml/hwbp_registers.xml

```diff
@@ -726,17 +732,17 @@
         </field>
 
         <field name="m" bits="9" access="WARL" reset="0">
-            When set, every instruction completed in or trap taken from
-            M-mode decrements \FcsrIcountCount by 1.
+            When set, enable this trigger in M-mode.
+        </field>
+        <field name="pending" bits="8" access="R/W" reset="0">
```

Comment:
Does this bit need to be writable?

Reply:
My thought was that an M-mode debugger might want to clear the bit for some reason, and surely somebody might have a clever use for setting it. I don't think it costs the hardware anything to make it writable given that it already exists, so I made it writable. If I'm wrong about hardware and making the bit writable makes implementations notably more complicated, I'm fine with it being read-only.